src/Marketplace/Marketplace.jsx

```diff
@@ -0,0 +1,318 @@
+import React, { useState, useEffect } from "react";
+import { getNativeByChain } from "helpers/networks";
+import { getCollectionsByChain } from "helpers/collections";
+import {
+  useMoralis,
+  useMoralisQuery,
+  useNewMoralisObject,
+} from "react-moralis";
+import { Card, Image, Tooltip, Modal, Badge, Alert, Spin } from "antd";
+import { useNFTTokenIds } from "hooks/useNFTTokenIds";
+import {
+  FileSearchOutlined,
+  RightCircleOutlined,
+  ShoppingCartOutlined,
+} from "@ant-design/icons";
+import { useMoralisDapp } from "providers/MoralisDappProvider/MoralisDappProvider";
+import { getExplorer } from "helpers/networks";
+import { useWeb3ExecuteFunction } from "react-moralis";
+const { Meta } = Card;
+
+const styles = {
+  NFTs: {
+    display: "flex",
+    flexWrap: "wrap",
+    WebkitBoxPack: "start",
+    justifyContent: "flex-start",
+    margin: "0 auto",
+    maxWidth: "1000px",
+    gap: "10px",
+  },
+  banner: {
+    display: "flex",
+    justifyContent: "space-evenly",
+    alignItems: "center",
+    margin: "0 auto",
+    width: "600px",
+    //borderRadius: "10px",
+    height: "150px",
+    marginBottom: "40px",
+    paddingBottom: "20px",
+    borderBottom: "solid 1px #e3e3e3",
+  },
+  logo: {
+    height: "115px",
+    width: "115px",
+    borderRadius: "50%",
+    // positon: "relative",
+    // marginTop: "-80px",
+    border: "solid 4px white",
+  },
+  text: {
+    color: "#041836",
```

Comment:
would like to see some of these settings (especially colors) moved to some kind of constants file eventually

Reply:
Absolutely will, just trying to get some stuff out for now so the other guys can work on it. I added emotion as well on my next fork